lib/inc/internal/facts/linux/fips_resolver.hpp

```diff
@@ -0,0 +1,31 @@
+/**
+ * @file
+ * Declares the Linux fips fact resolver.
+ */
+#pragma once
+
+#include "../resolvers/fips_resolver.hpp"
+
+namespace facter { namespace facts { namespace linux {
+
+    /**
+     * Responsible for resolving fips-related facts.
+     */
+    struct fips_resolver : resolvers::fips_resolver
+    {
+     // protected:
```

Comment:
This `protected` needs uncommenting (and the comment below can be indented a tad more)